### DISCUSSION TITLE
Question: How to update to latest xDrip+ from custom built version

### DISCUSSION THREAD

**Discussion by seattlebrighton** [NightscoutFoundation/xDrip#1985] [Q&A]
My custom-built version of xDrip+ (included mods for Amazfit band5) won't let me update it to a current nightly, due to different developer keys, I think. What's the best way to keep my settings but jettison the old build and update to the latest nightly?
Thanks, -Mark

**Comment by Navid200** [comment]
Backup the database:
https://navid200.github.io/xDrip/docs/Backup-Database.html

Uninstall (jettison :o)).

Install the official release.
Restore the database:
https://navid200.github.io/xDrip/docs/Restore-Database.html

**Reply by seattlebrighton** [reply]
Thanks Navid, Should I backup settings, too, or does the database backup handle that? 
Sorry, your link says that database backup also backs up settings. Thanks. I assume that uninstalling won't delete the database zip file that's been stored in internalStorage/xdrip? I'll copy that zip file to cloud just in case.

**Reply by Navid200** [reply]
When you backup the database, the settings are also backed up in a different file as explained.
When the time comes for restore, you can decide if you also want to restore the settings.

All your alert settings are a part of the database.

There are some settings, like Trust Auto-Connect, that are supposed to be set depending on the device brand.  So, it's not always a good idea to restore settings.

But, since you are not changing the phone, you can also restore the settings.
